tests/fracs/test_fracture_importer.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Please ensure the module docstring gives an overview of what is being tested here. This applies to all files.